caddy-gen/Caddyfile.example

```diff
@@ -0,0 +1,55 @@
+http://siyuan.sjtu.edu.cn {
+    redir / https://{host}{uri} 301
+}
+
+anaconda.siyuan.sjtu.edu.cn/ {
+    root /mnt/anaconda
+    browse
+}
+
+packagist.siyuan.sjtu.edu.cn/ {
+    root /mnt/packagist
+    browse
+}
+
+https://siyuan.sjtu.edu.cn {
+    file_server /putty/* browse
```

Comment:
We have two more functionalities to implement. First, we have to serve our frontend file to our users. Secondly, users should be able to access our lug endpoint. You could refer to current `Caddyfile` on master for how to implement this.